src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
 }
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
set `border-box` for all items in page, not only for the product. use `*` selector